nixos/lib/make-options-doc/generateDoc.py

```diff
@@ -49,14 +53,14 @@ def generate_commonmark(options):
         if 'default' in value:
             print('*_Default_*')
             print('```')
-            print(json.dumps(value['default'], ensure_ascii=False, separators=(',', ':')))
+            print(json.dumps(value['default'], cls=OptionsEncoder, ensure_ascii=False, separators=(',', ':')))
```

Comment:
This is still wrong. It shouldn't use a json encoder at all, but rather pretty print a Nix value.

Reply:
Ah, cool! I was worried there could be arbitrary JSON in there, given all these json dumps, but looks like we don't need any of that.